plugin/trino-mysql/src/main/java/io/trino/plugin/mysql/MySqlConfig.java

```diff
@@ -81,4 +82,21 @@ public MySqlConfig setDriverUseInformationSchema(boolean driverUseInformationSch
         this.driverUseInformationSchema = driverUseInformationSchema;
         return this;
     }
+
+    public Properties getConnectionProperties()
```

Comment:
Config classes should only contain config, not other code. Make this a utility method in `MySqlClientModule` or another location.

Reply:
Noted :) (And fixed)